va/va.h

```diff
@@ -523,6 +523,13 @@ typedef enum
      */
     VAConfigAttribEncQualityRange     = 21,
     /**
+     * \brief Encoding quantization attribute. Read-only.
+     *
+     * This attribute conveys whether the driver supports certain types of quantization methods
+     * for encoding (e.g. trellis).
```

Comment:
It would be better to add a comment that " See \c VA_ENC_QUANTIZATION_xxx for the list of quantization methods", otherwise a reader will have no idea about the quantization methods.